openstack_controller/datadog_checks/openstack_controller/openstack_controller.py

```diff
@@ -542,6 +544,9 @@ class OpenStackControllerCheck(AgentCheck):
     def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
 
+        if is_affirmative(init_config.get('trace_check', False)):
+            add_trace_check(self)
```

Comment:
This doesn't belong here, but rather in the base class `__init__`.

Reply:
This should go here. It adds the openStackControllerCheck to the list of checks that should be traced - https://github.com/DataDog/integrations-core/blob/4188e443c012669f49e8c6f8a29bcbc8df8d6c64/datadog_checks_base/datadog_checks/base/utils/tracing.py#L20